src/pages/Community/CommunityPage.tsx

```diff
@@ -224,6 +222,11 @@ const CommunityPage: FC<Props> = ({
         </Box>
       )}
       <Divider />
+      {isFeatureEnabled(FEATURE_COMMUNICATIONS_DISCUSSIONS) && (
+        <Link component={RouterLink} to={`${url}/discussions`}>
+          Discussions
```

Comment:
translation

Reply:
The PR is not about this. Will be fixed in other PR.